frontend/packages/console-dynamic-plugin-sdk/src/app/components/status/statuses.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { InProgressIcon } from '@patternfly/react-icons';
+import { StatusComponentProps } from '../../../extensions/console-types';
+import GenericStatus from './GenericStatus';
+import { RedExclamationCircleIcon, GreenCheckCircleIcon, BlueInfoCircleIcon } from './icons';
+
+export const ErrorStatus: React.FC<StatusComponentProps> = (props) => (
+  <GenericStatus {...props} Icon={RedExclamationCircleIcon} title={props.title} />
```

Comment:
we can let props go in the flow as they both have `title` unless we are not updating it, i don't think we need to pass title explictly 

```suggestion
  <GenericStatus {...props} Icon={RedExclamationCircleIcon}  />
```

Reply:
@invincibleJai it then does not comply with the Icon proptypes:
```
type GenericStatusProps = StatusComponentProps & {
  Icon: React.ComponentType<{ title?: string }>;
  title: string;
  popoverTitle?: string;
  noTooltip?: boolean;
};
```
`Type '{ Icon: FC<ColoredIconProps>; title?: string; iconOnly?: boolean; noTooltip?: boolean; className?: string; popoverTitle?: string; children?: ReactNode; }' is not assignable to type '{ Icon: ComponentType<{ title?: string; }>; ...`